### DISCUSSION TITLE
Problem with clock tweak

### DISCUSSION THREAD

**Discussion by mateusfn98** [ramensoftware/windhawk#46] [Q&A]
My clock customization was setted and ok, but i did something (idk if the new windows att change something) and my clock tweak doesn't open anymore.
Show me a dialog with loading and never activate.

**Comment by m417z** [comment]
Did you change the mod settings? If you configured the news feed incorrectly, perhaps it can't be fetched.
You can reset your mod settings by uninstalling and re-installing the mod.
